### PR TITLE
Added option for maintain or delete duplicate css properties

### LINKED ISSUE #76: Delete duplicate porperties
When combine it selectors is common get things like this.

.myClass {
  background: red;
  font-size: 15px;
  color: black;
  background: green;
}

The cascade takes de second background property, ¿is possible delete the first one?.

Thank you!

### PR DESCRIPTION
**Pull request by aldebaran798**
Proposal for resolve #76 issue

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -32,14 +32,39 @@ function sortGroups(selector) {
...
     sortGroups(selector);
   }
 );
 
-module.exports = postcss.plugin('postcss-combine-duplicated-selectors', () => {
+const defaultOptions = {
+  removeDuplicatedProperties: false
+};
+
+module.exports = postcss.plugin('postcss-combine-duplicated-selectors', (options = defaultOptions) => {
```
Comment (ChristianMurphy) [inline src/index.js]: I would lean using destructuring to access values from the options object.
``` js
(options = defaultOptions) => {
```
to
``` js
({removeDuplicatedProperties = true}) => {
```

File: test/duplicated-properties.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
+const test = require('ava');
+const testFactory = require('./_test-factory');
+const plugin = require('../src')({removeDuplicatedProperties: true});
```
Comment (ChristianMurphy) [inline test/duplicated-properties.js]: thanks for adding tests for the new deduplication functionality. :+1: 
Could some tests ensuring that properties are preserved when `removeDuplicatedProperties: false` be added as well?

File: src/index.js [hunk 3]
```diff
@@ -32,14 +32,39 @@ function sortGroups(selector) {
...
 }
 
+/**
+ * Remove duplicated properties
+ * @param {Object} selector - postcss selector node
+ */
+function removeDuplicatedProperties(selector) {
+  let actualNodeIndex = selector.nodes.length - 1;
+  // Remove duplicated properties from bottom to top
+  while (actualNodeIndex >= 1) {
```
Comment (ChristianMurphy) [inline src/index.js]: :art: 
```js
for (let actualNodeIndex = selector.nodes.length - 1; actualNodeIndex >= 1; actualNodeIndex--) {
```

File: src/index.js [hunk 4]
```diff
@@ -32,14 +32,39 @@ function sortGroups(selector) {
...
+    let beforeNodeIndex = actualNodeIndex - 1;
+    while (beforeNodeIndex >= 0) {
```
Comment (ChristianMurphy) [inline src/index.js]: :art: 
``` js
for (let beforeNodeIndex = actualNodeIndex - 1; beforeNodeIndex >= 1; beforeNodeIndex--) {

File: src/index.js (hunk 1 above)
Comment (aldebaran798) [inline src/index.js]: I tried this, but the linter crash

**Pull request merged**